### PR TITLE
chore: migrate to new prow reviewing

### PR DESCRIPTION
**Pull request by wuhuizuo**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

Add prow `OWNERS` files, it's a part of the implement of https://internals.tidb.io/t/topic/854.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.5 (TiDB Operator 1.5 versions)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [x] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- Backgroup: https://internals.tidb.io/t/topic/854
- Ref: https://github.com/ti-community-infra/configs/issues/824

### CODE REVIEW

**Comment by wuhuizuo** [comment]
It should be cherry-picked to release branches when merged, but no need to deploy.

**Review by lilin90 — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - glkappe
+  - GMHDBJD
+  - hfxsd
+  - Icemap
+  - Joyinqin
+  - KanShiori
+  - lucklove
+  - Oreoxmt
+  - tiancaiamao
+  - zimulala
```
Comment (lilin90) [inline OWNERS:41]:
```suggestion
  # docs-reviewers:
  - 3pointer
  - amyangfei
  - anotherrachel
  - crazycs520
  - dveeden
  - glkappe
  - GMHDBJD
  - Icemap
  - Joyinqin
  - KanShiori
  - lucklove
  - tiancaiamao
  - zimulala
```

**Pull request merged**